app/models/channels/twilio.rb

```diff
@@ -19,11 +19,14 @@ class Channels::Twilio < Channel
   config_accessor :account_sid
   config_accessor :auth_token
   config_accessor :number
+  config_accessor :base_url
 
   attr_protected :guid
 
   before_create :create_guid
 
+  validates :base_url, format: URI::regexp(["http", "https"]), allow_nil: true, allow_blank: true
```

Comment:
This validation is wrong. `aahttpbb` will match the regex.
Either change the regex to `/\Ahttps?:\/\/` to ensure a prefix as intended.
Or define a custom validator that will use URI.parse as in https://stackoverflow.com/a/7167988/30948

Reply:
`URI::regexp` is fine oops